### PR TITLE
Add Support for PHP 8.4

### PR DESCRIPTION
**Pull request by gsteel**

### CODE REVIEW

**Review by Ocramius — APPROVED** [review]

**Comment by Ocramius** [comment]
BTW, worth trying on 2.x?

File: .laminas-ci.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "backwardCompatibilityCheck": true,
     "ignore_php_platform_requirements": {
-        "8.3": false
+        "8.4": true
```
Comment (froschdesign) [inline .laminas-ci.json:4]: This can be removed if all required components also support PHP in version 8.4?

Comment (gsteel) [reply .laminas-ci.json:4]: Once support is available in stdlib etc, yes but it's needed until then

Comment (froschdesign) [reply .laminas-ci.json:4]: Thanks for the clarification!

Comment (froschdesign) [reply .laminas-ci.json:4]: Do we have a process that checks this or do we have to go through everything manually later?

Comment (gsteel) [reply .laminas-ci.json:4]: We have to go through everything manually 😢

**Review by froschdesign — APPROVED** [review]

**Pull request merged**